# key for defining mpc opt variables
    float32[] zeta0
    float32[] u0
    float32 sim_time